mythril/src/acpi/seabios.rs

```diff
@@ -0,0 +1,175 @@
+use crate::error::{Error, Result};
```

Comment:
I'd like all new modules going forward to require docs for everything, so we can eventually just require it for the whole project. It would probably also be good to write a bit of a module description given that you had to kind of reverse-engineer this from seabios, right?

Reply:
Yeah I'll add module level docs for that with the protocol described. Everything in the acpi is `deny(missing_docs)`, so everything in here should be documented, but some documentation should probably be improved (like the module level docs).